contributors.txt

```diff
@@ -139,4 +139,5 @@ YYYY/MM/DD, github id, Full name, email
 2020/09/09, adamshakhabov, Adam Shakhabov, theadamo86(at)gmail.com
 2020/09/13, hoborm, Mihaly Hobor, mail(at)hobor.xyz
 2020/07/11, blitzmann, Ryan Holmes, holmes.ryan.90(at)gmail.com
-2020/10/04, GintasS, Gintautas Švedas, svedgintas(at)gmail.com
\ No newline at end of file
+2020/10/04, GintasS, Gintautas Švedas, svedgintas(at)gmail.com
+2020/10/06, vrubleg, Evgeny Vrublevsky, me@veg.by
```

Comment:
You may want to replace the `@` sign :)

Reply:
As for me, using `(at)` makes no sense, because all crawlers which collect emails already know about popular substitutes of `@` and `.`. Spam filters are really good nowadays, so I always post my emails as is. One of my emails is public for almost 20 years, it is included probably in all possible spam lists. It was an issue 15 years ago, 20+ unfiltered spam messages per day, but now it is just a message in a month or so.

I (or you) can replace it to `(at)` for consistency with others if you wish, but there is another person who used `@` as is in their email also in the list.

Reply:
No worries, wanted to make sure you did it consciously. 
...And our emails are public in the git history too :)